Makefile

```diff
@@ -30,3 +29,23 @@ doc:
 	find /tmp/goxel_src -name '*.c' | xargs rm
 	mkdir -p doc ndconfig
 	naturaldocs -i /tmp/goxel_src -o html doc -p ndconfig
+
+PREFIX ?= /usr/local
+
+.PHONY: install
+install:
+	install -Dm744 goxel $(DESTDIR)$(PREFIX)/bin/goxel
+	for size in 16 24 32 48 64 128 256; do \
+		install -Dm644 data/icons/icon$$size.png $(DESTDIR)$(PREFIX)/share/icons/hicolor/$${size}x$$size/apps/goxel.png; \
+	done
```

Comment:
Run `gtk-update-icon-cache -ftq $(DESTDIR)$(PREFIX)/share/icons/hicolor`

Reply:
Is this also required for KDE?

Reply:
No, KDE has its own caching mechanism.